app/policies/article_policy.rb

```diff
@@ -0,0 +1,34 @@
+class ArticlePolicy < ActionPolicy::Base
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
```

Comment:
Лучше в новых классах наследоваться от ApplicationPolicy, а initialize не нужен

```suggestion
class ArticlePolicy < ApplicationPolicy
```